ractor.rb

```diff
@@ -38,38 +187,75 @@ def self.new(*args, name: nil, &block)
     __builtin_ractor_create(loc, name, args, b)
   end
 
-  # return current Ractor
+  # Returns the currently executing Ractor.
+  #
+  #   Ractor.current #=> #<Ractor:#1 running>
   def self.current
     __builtin_cexpr! %q{
       rb_ec_ractor_ptr(ec)->self
     }
   end
 
+  # Returns total count of Ractors currently running.
+  #
+  #    Ractor.count                   #=> 1
```

Comment:
I believe that the way it currently shows the point better (save for `sleep`, which I'll address below): with `Ractor.count` inside other ractor it is not clear: whether it would be `2` only _inside_ the second ractor, or throughout the system, while the second ractor is alive.